spec/checkWipDraftPRSpec.js

```diff
@@ -66,8 +66,15 @@ describe('Oppiabot\'s', () => {
     done();
   });
 
-  describe('WIP PRs', () => {
+  describe('WIP PRs without skip prefix', () => {
     beforeEach((done) => {
+      github.git = {
+        getCommit: jasmine.createSpy('getCommit').and.resolveTo({
+          data: {
+            message: 'changes'
```

Comment:
Make it more relevant like `commit withou skip prefix`

Reply:
Done.